samples/online-boutique/kubernetes-manifests/deployments/emailservice.yaml

```
@@ -33,26 +33,37 @@ spec:
         app: emailservice
     spec:
       serviceAccountName: email
+      terminationGracePeriodSeconds: 5
+      securityContext:
+        fsGroup: 1000
+        runAsGroup: 1000
+        runAsNonRoot: true
+        runAsUser: 1000
       containers:
       - name: server
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+              - ALL
+          privileged: false
+          readOnlyRootFilesystem: true
         image: gcr.io/google-samples/microservices-demo/emailservice:v0.3.6
```

Don't we need to update the version to 0.9.0?

---

Whoops! Thank you catching that. I really appreciate your thoroughness. 
I will update:
```suggestion
        image: gcr.io/google-samples/microservices-demo/emailservice:v0.9.0
```